src/modal/Modal.jsx

```diff
@@ -3,6 +3,11 @@ import ReactDOM from 'react-dom';
 import PropTypes from 'prop-types';
 import './Modal.css';
 
+const Keys = {
+    Tab: 9,
+    Esc: 27,
+
+}
```

Comment:
There are other keycodes also used like for `space` and `enter`. Can you please add those as well?

Reply:
Added missed keycodes.